more/free-programming-cheatsheets.md

```diff
@@ -36,6 +37,21 @@
 * [Tensorflow](#tensorflow)
 
 
+### APL
+
+* [A reference card for GNU APL](https://github.com/jpellegrini/gnu-apl-refcard/blob/master/aplcard.pdf) - jpellegrini (PDF)
+* [CheatSheet - Binding Strengths](https://docs.dyalog.com/latest/CheatSheet%20-%20Binding%20Strengths.pdf) - Dyalog (PDF)
```

Comment:
maybe there's one link for the dyalog cheatsheets?

Reply:
There's only https://docs.dyalog.com/latest which probably isn't something you want to link to.

Reply:
@eshellman Is this acceptable as-is?

Reply:
how about https://www.dyalog.com/documentation_180.htm#CHEAT ?

Reply:
That's version-specific and will be out of date within a month. Is https://docs.dyalog.com#CHEAT acceptable even though it is a redirect?

Reply:
yes, that looks good